src/main/java/bio/terra/common/logging/RequestCacheFilter.java

```diff
@@ -28,4 +29,10 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
     chain.doFilter(reqWrapper, resWrapper);
     resWrapper.copyBodyToResponse();
   }
+
+  @Override
```

Comment:
do these overrides serve to keep the filters alive forever? Seems odd you'd need to override with empty bodies.